### PR TITLE
ARROW-12838: [Java][Gandiva] Fix JNI CI test

### PR DESCRIPTION
**Pull request by jvictorhuguenin**
Fix checkstyle error in JNI CI build

### CODE REVIEW

**Comment by kou** [comment]
It seems that there are still 3 failures:

https://github.com/apache/arrow/pull/10367/checks?check_run_id=2635803647#step:8:11064

```text
Error:  Tests run: 85, Failures: 3, Errors: 0, Skipped: 2
```

Could you confirm them?

**Comment by jvictorhuguenin** [comment]
@kou, I fixed them, one of the tests was considering that null is equal to null in the IN expression query and having an Assertion error

File: java/gandiva/src/test/java/org/apache/arrow/gandiva/evaluator/ProjectorTest.java [hunk 1]
```diff
@@ -1193,7 +1193,7 @@ public void testInExpr() throws GandivaException, Exception {
     Field c1 = Field.nullable("c1", int32);
 
     TreeNode inExpr =
-        TreeBuilder.makeInExpressionInt32(TreeBuilder.makeField(c1), Sets.newHashSet(1, 2, 3, 4, 5, 15, 16));
+            TreeBuilder.makeInExpressionInt32(TreeBuilder.makeField(c1), Sets.newHashSet(1, 2, 3, 4, 5, 15, 16));
```
Comment (kou) [inline java/gandiva/src/test/java/org/apache/arrow/gandiva/evaluator/ProjectorTest.java:1196]: Is this indent style change intentional?
It seems that we use 4 spaces not 8 spaces for this case in other places.

Comment (jvictorhuguenin) [reply java/gandiva/src/test/java/org/apache/arrow/gandiva/evaluator/ProjectorTest.java]: It was unintentional I'm sorry, my IDE was with a bad configuration, already fixed it.

**Review by kou — APPROVED** [review]
+1

**Pull request closed without merging**